README.rst

```diff
@@ -1,64 +1,26 @@
 Currentscape
 ============
 
-.. raw:: html
-
-	<table>
-	<tr>
-	  <td>Latest Release</td>
-	  <td>
-	    <a href="https://pypi.org/project/currentscape/">
-	    <img src="https://img.shields.io/pypi/v/currentscape.svg" alt="latest release" />
-	    </a>
-	  </td>
-	</tr>
-	<tr>
-	  <td>Documentation</td>
-	  <td>
-	    <a href="https://currentscape.readthedocs.io/">
-	    <img src="https://readthedocs.org/projects/currentscape/badge/?version=latest" alt="latest documentation" />
-	    </a>
-	  </td>
-	</tr>
-	<tr>
-	  <td>License</td>
-	  <td>
-	    <a href="https://github.com/BlueBrain/Currentscape/blob/main/LICENSE.txt">
-	    <img src="https://img.shields.io/pypi/l/currentscape.svg" alt="license" />
-	    </a>
-	</td>
-	</tr>
-	<tr>
-	  <td>Build Status</td>
-	  <td>
-	    <a href="https://github.com/BlueBrain/Currentscape/actions">
-	    <img src="https://github.com/BlueBrain/Currentscape/workflows/Test/badge.svg?branch=main" alt="Actions build status" />
-	    </a>
-	  </td>
-	</tr>
-	<tr>
-	  <td>Coverage</td>
-	  <td>
-	    <a href="https://codecov.io/gh/BlueBrain/currentscape">
-	    <img src="https://codecov.io/github/BlueBrain/Currentscape/coverage.svg?branch=main" alt="coverage" />
-	    </a>
-	  </td>
-	</tr>
-	<tr>
-		<td>Gitter</td>
-		<td>
-			<a href="https://gitter.im/bluebrain/currentscape">
-			<img src="https://badges.gitter.im/Join%20Chat.svg"
-		</a>
-		</td>
-	</tr>
-	</table>
+
++----------------+------------+
```

Comment:
Does this work in rst files?

Reply:
Does it work in the doc though? I remember Alexander put this complex thing to make things work in sphinx, but maybe that was not necessary.

Reply:
I just checked, it works also in the doc. I noticed that we weren't using readme in the docs, so I added it, with a few more fixes.